en/micro-integrator/docs/references/connectors/exporting-artifacts.md

```diff
@@ -0,0 +1,27 @@
+# Exporting artifacts
+In this document we focus on creating the Connector Exporter Project and Exporting the CApp. 
```

Comment:
In this document --> In this section (as we are going to make a compiled doc using this page section will make more sense).

Reply:
In this document we focus on creating the Connector Exporter Project and Exporting the CApp. CApp is the deployable artefact on the Enterprise Integrator runtime.

Reply:
Done